modules/luci-mod-network/htdocs/luci-static/resources/view/network/wireless.js

```diff
@@ -938,7 +971,7 @@ return view.extend({
 				o.inputtitle = isDisabled ? _('Enable') : _('Disable');
 				o.onclick = ui.createHandlerFn(s, network_updown, s.section, s.map);
 
-				o = ss.taboption('general', CBIWifiFrequencyValue, '_freq', '<br />' + _('Operating frequency'));
+				o = ss.taboption('general', CBIWifiFrequencyValue, '_freq', '<br />' + _('Operating frequency'), _('Some channels may be restricted to Indoor Only use. Make sure to follow what your Regulatory Domain advise if a channel is reported as follow.'));
```

Comment:
Maybe better: "Some channels may be restricted to Indoor Only use by your Regulatory Domain. Make sure to follow this advise if a channel is reported as such."